src/Engines/Processors/shaderProcessor.ts

```diff
@@ -23,7 +24,10 @@ export class ShaderProcessor {
     public static Process(sourceCode: string, options: ProcessingOptions, callback: (migratedCode: string) => void) {
         this._ProcessIncludes(sourceCode, options, (codeWithIncludes) => {
             let migratedCode = this._ProcessShaderConversion(codeWithIncludes, options);
-            callback(migratedCode);
+            let sci = new ShaderCodeInliner(migratedCode);
+            sci.debug = true;

```

Comment:
do we want to keep it in debug?

Reply:
we wont even keep it in GL :-), nw I flag as resolve as this file will not be in the final PR.